src/pybind/mgr/dashboard/tests/test_controllers.py

```diff
@@ -175,7 +175,7 @@ def test_rest_collection(self):
         self.assertStatus(200)
         self.assertJsonBody({'key': '300', 'data': 30})
 
-    def test_rest_resourse(self):
+    def test_rest_resource(self):
```

Comment:
Nit: please put this change in separate commit and explicitly mention it in the PR description. Also, it seems like you only rename the method, but no occurrence of where it is actually called?

Reply:
@LenzGr
> Also, it seems like you only rename the method, but no occurrence of where it is actually called?

That's fine as it's a test method and doesn't need to be called explicitly.